spec/reader_spec.rb

```diff
@@ -126,6 +131,11 @@
     it "returns a single page from no_text_spaces" do
       expect(PDF::Reader.new(no_text_spaces).page(1)).to be_a_kind_of(PDF::Reader::Page)
     end
+
+    it "raises InvalidPageError when an invalid page number is requested" do
+      reader = PDF::Reader.new(pdf_spec_file("cairo-basic"))
+      expect { reader.page(10) }.to raise_error(PDF::Reader::InvalidPageError)
+    end
```

Comment:
These specs look good, but can the first one be duplicated into `spec/integration_spec.rb`? As a file-parsing lib, I rely on that set of integration specs across the growing corpus of test files to avoid regressions.

Reply:
👍